static/templates/settings/organization_settings_admin.hbs

```diff
@@ -1,6 +1,24 @@
 <div id="organization-settings" data-name="organization-settings" class="settings-section">
     <form class="form-horizontal admin-realm-form org-settings-form">
 
+        <div id="org-pub-listing" class="org-subsection-parent">
```

Comment:
I think this feature belongs organizations in the "organization profile" page.

Reply:
yes, that make sense! made the change..